### PR TITLE
Add uuid field to Origin

### PR DESCRIPTION
**Pull request by BrianChapman**

### CODE REVIEW

File: guardian/testapp/migrations/0010_auto_20170628_1735.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```
Comment (fgaudin) [inline guardian/testapp/migrations/0010_auto_20170628_1735.py:1]: is this migration related?

**Review by fgaudin — APPROVED** [review]

File: guardian/testapp/migrations/0010_auto_20170628_1735.py (hunk 1 above)
Comment (BrianChapman) [reply guardian/testapp/migrations/0010_auto_20170628_1735.py:1]: Django always creates that migration. You can run it over and over and it will keep creating it. Should I just remove it from this PR?
